flux-driver/src/callbacks.rs

```diff
@@ -69,40 +69,45 @@ impl Callbacks for FluxCallbacks {
 }
 
 fn check_crate(tcx: TyCtxt, sess: &FluxSession) -> Result<(), ErrorGuaranteed> {
-    let mut specs = SpecCollector::collect(tcx, sess)?;
+    tracing::info_span!("check_crate").in_scope(|| {
```

Comment:
Not sure if I set things up wrong, but I don't see this in the output when I run it on a sample repo.

Also, I wasn't sure whether it would make more sense to use the `#[instrument]` macro here.